source/uwp/Renderer/idl/AdaptiveCards.XamlCardRenderer.idl

```diff
@@ -777,6 +793,7 @@ namespace AdaptiveCards
         interface IAdaptiveActionEventArgs : IInspectable
         {
             [propget] HRESULT Action([out, retval] IAdaptiveActionElement** action);
+            [propget] HRESULT Inputs([out, retval] HSTRING* value);
```

Comment:
I believe the reasoning behind returning json for the submit inputs is in order to make it easier to send the results back to a theoretical server (in say, the bot case).  The other renderers return json as well.  

@matthidinger can speak to this further and comment on whether we might additionally want to add something more convenient for local callers.

---
In reply to: [118375602](https://github.com/Microsoft/AdaptiveCards/pull/369#discussion_r118375602) [](ancestors = 118375602)